Formula/x/xgboost.rb

```diff
@@ -5,6 +5,11 @@ class Xgboost < Formula
   sha256 "8f909899f5dc64d4173662a3efa307100713e3c2e2b831177c2e56af0e816caf"
   license "Apache-2.0"
 
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    strategy :github_latest
```

Looking at the latest version, the v3.0.2 release on GitHub was created a little over an hour after the tag and this formula is using a GitHub release asset, so I think it's fine to use the `GithubLatest` strategy here.